homeassistant/components/alexa/handlers.py

```diff
@@ -520,21 +521,49 @@
     """Process an adjust volume step request."""
     # media_player volume up/down service does not support specifying steps
     # each component handles it differently e.g. via config.
-    # For now we use the volumeSteps returned to figure out if we
-    # should step up/down
-    volume_step = directive.payload["volumeSteps"]
+    # This workaround will simply call the volume up/Volume down the amount of steps asked for
+    # When no steps are called in the request, Alexa sends a default of 10 steps which for most
+    # purposes is too high.  A device_state_attribute can be set in the media player to set this to a new default
+    # The default without the attribute is set 1 in this case.
     entity = directive.entity
-
-    data = {ATTR_ENTITY_ID: entity.entity_id}
-
-    if volume_step > 0:
-        await hass.services.async_call(
-            entity.domain, SERVICE_VOLUME_UP, data, blocking=False, context=context
-        )
-    elif volume_step < 0:
-        await hass.services.async_call(
-            entity.domain, SERVICE_VOLUME_DOWN, data, blocking=False, context=context
-        )
+    volume_int = int(directive.payload["volumeSteps"])
+    is_default = bool(directive.payload["volumeStepsDefault"])
+    default_steps = 1
+    sleep_delay = 0.5
+
+    if "volume_steps_default" in entity.attributes:
+        try:
+            default_steps = int(entity.attributes["volume_steps_default"])
```

Comment:
We should not introduce new attributes to an integration for specific use cases. We like to put a little more thought into the attributes that we support. Please remove this.

You can open an architecture issue if you want to propose a new attribute for an entity. I don't think this warrants the addition of a newly supported attribute though.

Reply:
It's just a nice to have but definitively not a required one as all one has to say is "Alexa, turn up the volume by 5 on tv" instead of just 'Alexa, turn up the volume in tv".   
There are a few additional things that can be added to the media_player as suggested by others for more specific control of various hardware but as you noted in this case, all functionality can be accomplished by existing attributes.  I've remove it and kept it at basics. Thanks.